src/app/components/wrapper-components/local-game-wrapper/local-game-wrapper.component.ts

```diff
@@ -130,22 +129,19 @@ export class LocalGameWrapperComponent extends GameWrapper<string> implements Af
     }
 
     public async proposeAIToPlay(): Promise<void> {
-        if (await this.hasSelectedAI()) {
+        const isAISelected: boolean = await this.hasSelectedAI();
+        await this.setInteractive(isAISelected === false);
+        if (isAISelected) {
             // It is AI's turn, let it play after a small delay
-            this.gameComponent.setInteractive(false);
-            await this.updateBoard(false);
-            const playingAI: MGPOptional<{ ai: AbstractAI, options: AIOptions }> = this.getPlayingAI();
+            const playingAI: MGPOptional<{ ai: AbstractAI, options: AIOptions }> = this.getCurrentAIOption();
             if (playingAI.isPresent()) {
                 window.setTimeout(async() => {
                     await this.doAIMove(playingAI.get().ai, playingAI.get().options);
+                    this.cdr.detectChanges();
```

Comment:
nécessaire lui ?

Reply:
Oui sinon render pas le last move de l'ia :p

Reply:
ptit `// triggers the rendering of AI move` à rajouter alors